common/web/lm-worker/build-wrap-and-minify.js

```diff
@@ -65,11 +65,16 @@ const srcMapString = `//# sourceMappingURL=data:application/json;charset=utf-8;b
  * but my attempts to do so end up triggering errors when loading.
  */
 
+let rawScript = workerConcatenation.script.toString();
+// Two layers of encoding:  one for the raw source (parsed by the JS engine),
+// one to 'unwrap' it from a string _within_ that source.
+let jsonDoubleEncoded = JSON.stringify(JSON.stringify(rawScript));
```

Comment:
Thinking should be able to single-encode this? Can you check? That would then remove the `JSON.parse` call in unwrap.ts as well.

Reply:
That does indeed work; I stand corrected.